docs/usage/other_features/zeppelin_context.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: page
+title: "Zeppelin-Context"
+description: "The Zeppelin-Context is a system-wide container for a variety of user-specific settings and parameters that are accessible across notebooks, cells, and interpreters."
+group: usage/other_features
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+
+# Zeppelin-Context
+
+<div id="toc"></div>
+
+The zeppelin-context is a system-wide container for common utility functions and
+user-specific data. It implements functions for data input, data display, etc. that are
+often needed but are not uniformly available in all interpreters.
+Its single per-user instance is accessible across all of the user's notebooks and cells,
+enabling data exchange between cells - even in different notebooks.
+But the way in which the zeppelin-context is used, and the functionality available differs
+depending on whether or not the associated interpreter is based on a programming language.
+Details of how the zeppelin-context is used for different purposes and in different
+environments is described below.
+
+## Usage in Programming Language Cells
+
+In many programming-language based interpreters (e.g. Spark, Python, R) the zeppelin-context is available
+as a predefined variable `z` that can be used by directly invoking its methods.
+The methods available on the `z` object are described below.
+Interpreters based on programming languages like spark.dep, beam, etc. also provide the
+predefined variable `z`.
+
+### Exploring Spark DataFrames
+The zeppelin-context provides a `show` method, which, using Zeppelin's `table` feature,
+can be used to nicely display a Spark DataFrame:
```

Comment:
this was in the Spark interpreter page, should we add back the context here, "In the Spark interpreter, the zeppelin context"...

Reply:
Done. 
Also added a futuristic line saying `show()` will be progressively available in other interpreters.